contracts/smart-account/modules/AccountRecoveryModule.sol

```diff
@@ -34,11 +34,13 @@ contract AccountRecoveryModule is
     string public constant NAME = "Account Recovery Module";
     string public constant VERSION = "0.1.0";
 
-    // keccak256(abi.encodePacked("ACCOUNT RECOVERY GUARDIAN SECURE MESSAGE"))
-    bytes32 public constant CONTROL_HASH =
-        0x1c3074d7ce4e35b6f0a67e3db0f31ff117ca8f0a1853e86a95837fea35af828b;
-    bytes32 public immutable controlHashEthSigned =
-        CONTROL_HASH.toEthSignedMessageHash();
+    // execute(address,uint256,bytes)
+    bytes4 public constant EXECUTE_SELECTOR = 0xb61d27f6;
+    // execute_ncC(address,uint256,bytes)
+    bytes4 public constant EXECUTE_OPTIMIZED_SELECTOR = 0x0000189a;
+    // Hash to be signed by guardians to make a guardianId
```

Comment:
wouldn't hardcoding these values make this module incompatible with other SmartAccounts? Should we stick to this or make these variables "immutable" and set them in the constructor?

Reply:
Fair point. I can do this. However, in any case any SA with other selectors will require to redeploy the module with own parameters then.

I think this issue won't be an issue, when SA execution selectors will be standardized (via 6900 or some other eip)